src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/RedundantImportCheck.java

```diff
@@ -48,6 +48,15 @@
  * &lt;module name="RedundantImport"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import java.util.*; // OK
+ * import java.io.*; // OK
+ * import java.util.Scanner; // violation, Scanner class included in java.util package
+ * import java.io.File; // violation, File class included in java.io package
```

Comment:
please add whole file content as it matter.
to make it visible to user that class that is imported is not used in whole file and can be removed safely.